src/IIIFPresentation/API/Features/Common/Helpers/ErrorHelper.cs

```diff
@@ -74,6 +74,21 @@ public static ModifyEntityResult<TCollection, ModifyCollectionType> ParentMustBe
         where TCollection : JsonLdBase
         => ModifyEntityResult<TCollection, ModifyCollectionType>.Failure("The parent must be a storage collection",
             ModifyCollectionType.ParentMustBeStorageCollection, WriteResult.Conflict);
+    
+    public static ModifyEntityResult<TCollection, ModifyCollectionType>? InvalidCanvasId<TCollection>(string? canvasId) 
+        where TCollection : JsonLdBase
+        => ModifyEntityResult<TCollection, ModifyCollectionType>.Failure($"The canvas ID {canvasId} is invalid",
+            ModifyCollectionType.InvalidCanvasId, WriteResult.BadRequest);
+
+    public static ModifyEntityResult<TCollection, ModifyCollectionType>? DuplicateCanvasId<TCollection>(string? canvasId)
+        where TCollection : JsonLdBase
+        => ModifyEntityResult<TCollection, ModifyCollectionType>.Failure($"The canvas ID {canvasId} cannot be a duplicate",
+            ModifyCollectionType.DuplicateCanvasId, WriteResult.BadRequest);
+
+    public static ModifyEntityResult<TCollection, ModifyCollectionType>? CanvasOrderDifferentCanvasId<TCollection>(string? canvasId)
+        where TCollection : JsonLdBase
+        => ModifyEntityResult<TCollection, ModifyCollectionType>.Failure($"The canvas ID {canvasId} must be the same within a canvas order",
```

Comment:
Not sure how but I think this error message could be worded to explain what's happening better